app/_landing_pages/insomnia/authentication-authorization.yaml

```diff
@@ -0,0 +1,94 @@
+metadata:
+  title: Authentication and authorization at Insomnia
+  breadcrumbs:
+    - /insomnia/
+  description: 
+  tags:
+    - authentication
+    - authorization
+    - whitelist
+
+rows:
+  - header:
+      type: h1
+      text: "Authentication and authorization at Insomnia"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: "Secure Insomnia by setting up native or external authentication and configure authorization settings."
+  
+  - header:
+      type: h2
+      text: "Authentication"
+    columns: 
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    There are two different types of authentication in Insomnia:
+                - type: text
+                  text: |
+                    - **External authentication (SSO) or native authentication:** Allows you to log in to the Insomnia app.
+                    - **Request authentication:** Allows you to configure authentication strategies when you send a request.
```

Comment:
It seems weird to me to have both of these on the same page, since they are completely different use cases

Reply:
I think it could stay here, but we have to rewrite this. I'll take that on.